configs/ssg_platform_tools-vfxplatform-eln.yaml

```diff
@@ -0,0 +1,50 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: vfxplatform
+  description: The VFX Reference Platform https://vfxplatform.com/
+  maintainer: sst_pt_libraries
+
+  packages:
+  # The minimum development environment is this:
+  - gcc
+  - gcc-c++
+  - binutils
+  - glibc
+  - glibc-common
+  - glibc-devel
+  - libstdc++
+  - libstdc++-devel
+  - libgcc
+  # Full development includes additional packages:
+  - glibc-utils
+  - glibc-static
+  - glibc-nss-devel
+  - libstdc++-static
+  # Support all locales.
+  - glibc-all-langpacks
+  # Packages used by VFX, see https://vfxplatform.com/
+  - python3
+  - python3-devel
+  - python3-numpy
+  - python3-alembic
+  - qt6-devel
+  - boost
+  - boost-devel
+  - tbb
+  - tbb-devel
```

Comment:
OK. Yes for ELN we are keeping tbb (ACG CL2).

Reply:
In that case please revert the unwanted listing thereof (see #822 and #841)